### DISCUSSION TITLE
Why doesn't hubs / admin work?

### DISCUSSION THREAD

**Discussion by kouzzy** [mozilla/hubs#5178] [Hubs Cloud Questions]
Hi all.
When the installation was completed, I logged in and looked into the room, it was pitch black.
When I enter the URL (.pack) on the "import content" screen of "hubs / admin" and press the "preview import" button, there is no response. Is this natural?
Also, is it normal to get a "server communication error" in all when pressing other buttons?

**Comment by pattersonbl2** [comment]
Can you send a screenshot with the errors included?

**Reply by kouzzy** [reply]
Thank you for your help.
this is "server communication error."
![servercommunicationerror](https://user-images.githubusercontent.com/55646710/155656302-dce1e4b7-82d6-4c3b-bbac-1d049e6d4e26.png)

and, press the "preview import" button, there is no response.
![nothrough](https://user-images.githubusercontent.com/55646710/155656431-952f4472-c277-4059-8e12-4fcffb1350ed.png)

and, I logged in and looked into the room, it was pitch black.
![blackspace](https://user-images.githubusercontent.com/55646710/155656485-7db1a90c-80f2-496c-a7a2-9e2fe07ee8d2.png)
